qiling/core.py

```diff
@@ -48,15 +48,11 @@ def __init__(
             filter = None,
             stop_on_stackpointer = False,
             stop_on_exit_trap = False,
-            stdin=None,
-            stdout=None,
-            stderr=None,
+            **kwargs
```

Comment:
Personally, I prefer `stdin`, `stdout`, `stderr` to be passed as arguments, since it looks more intuitive to me.

Reply:
That is debateable, I suppose.
Qiling constructor has too many arguments, of which some are quite obscure (e.g. log filtering, etc.). My goal is to simplify the constructor arguments as much as I can, and let the user assign directly whatever properties they may like to use (like the OS standard streams).

In the same manner, the `**kwargs` is there to "throw aside" all the obscure switches and not let them pollute the constructor... though I agree it might not be a super-elegant way to do that.

Reply:
Fine, make sense.